.github/workflows/lint-squid.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
           key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
 
       - name: install packages
-        run: corepack enable && yarn install --immutable
```

Comment:
cannot comment above but instead of L20 we should use setup-node and provider nodejs version and cache

Reply:
sounds good, I added npm cache in [4a55656](https://github.com/ChainSafe/Multix/pull/239/commits/4a5565682508f5484a03a9361569effc1a757a0f)